### PR TITLE
[WIP] Test releationships

### PR DESCRIPTION
**Pull request by rrasouli**
Testing openstack infra provider releationships section, all relationships methods.
Most of default values are greater than 0 or as default deployment values
We should consider a better compare than hard coded values.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 3479bc3200559e9f5696f668af605b730a3030fb:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 348c73a354fc8e389f42610b8552eff2de6778a9:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 617886773f35aaa4a49c588a58749a027d101c07:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 19593ca9541ee0d93ecbd42ea5b5162d0dbdf4ed:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit eefe8ed1822d6029502cbf9317c7d5a0a0c71b99:

Everything seems all right from lint perspective :smile: .

Commit inspection report:
* 75275fecea6fe84cde89e649b9437c337f4eecb4
  * Merge commit detected!!! :bangbang:
* d1d686b23bcc93f4d079dc5b773b58040c8dc733
  * Merge commit detected!!! :bangbang:
* 683d5a7fb0668b373b90f49bf2efedd4841afed1
  * Merge commit detected!!! :bangbang:
**The PR might not be acceptable in this state** :rage4:.

*CFME QE Bot*

**Comment by RonnyPfannschmidt** [comment]
@rrasouli can you rid of the merges, its a pain, please use rebase

File: cfme/tests/infrastructure/test_host_role_type.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import pytest
```
Comment (psav) [inline cfme/tests/infrastructure/test_host_role_type.py]: Please re order these imports inline with the guidelines

**Comment by psav** [comment]
Looks reasonable, please lets get it squashed down without the merges and we could merge this I think

**Pull request closed without merging**